### ISSUE TITLE
Warning when installing feedparser dependency via pip

### ISSUE THREAD

**Issue by ksurl** [Flexget/Flexget#2775]
### Expected behaviour:

no warnings

### Actual behaviour:

warning about yanked package version

### Steps to reproduce:
- Step 1: 
`pip install flexget`

#### Log:
<details>
 <summary>(click to expand)</summary>

```
WARNING: The candidate selected for download or install is a yanked version: 'feedparser' candidate (version 6.0.1 at https://files.pythonhosted.org/packages/2c/84/df6de99cba01afc82344c9cb3a79df100a00ac33396120f8aa66c72f0d84/feedparser-6.0.1-py2.py3-none-any.whl#sha256=c9b436334258e718e224c5ef8a78088557e0d0382506360f702de10667ffde32 (from https://pypi.org/simple/feedparser/) (requires-python:>=3.6))
Reason for being yanked: Wheels allow py2 installation despite being unsupported
```
</details>

### Additional information:

- FlexGet version: 3.x
- Python version: 3.9 (probably will be there for 3.6+)
- Installation method: pip
- Using daemon (yes/no): n/a
- OS and version: Ubuntu 18.04

### Suggestion:

Update requirements.txt to bump feedparser from 5.2.1 to 6.0.2. 6.0 and 6.0.1 are yanked. 6.x added support for python 3 and since flexget 3.x requires python 3, I don't see any issue doing this.

There is also an open [issue](https://github.com/pypa/pip/issues/8262) for pip which may indicate this is a bug.

**Comment by ksurl** [comment]
there is also a error for `six`. requirements says >=1.1.0 and the latest version is 1.15.0 but the error says flexget requires 1.13.0?

`flexget 3.1.84 requires six==1.13.0, but you'll have six 1.15.0 which is incompatible.`

this seems to be a conflict with pysftp which is required for the sftp_download/list plugins. Though I could be mistaken if it is a dependency for transmissionrpc or subdependency of 1 of those 2

**Comment by gazpachoking** [comment]
Fixed by #2782
